app/views/complaints/show.html.erb

```diff
@@ -7,17 +7,35 @@
   <div class="grid-row grid-gap">
     <div class="grid-col-4">
       <div class="summary-section">
-        <h2>Summary</h2>
-        <p>
-          <strong><%= @complaint.grantee %>:</strong>
-          <%= @complaint.summary %>
-        </p>
+        <div class="ct-section-divider">
+          <h2>Summary</h2>
+          <p>
+            <strong><%= @complaint.grantee %>:</strong>
+            <%= @complaint.summary %>
+          </p>
+        </div>
+        <% unless @tta_reports.empty? %>
+          <%= render "tta_report_list", :tta_reports => @tta_reports %> 
```

Comment:
[suggestion / nitpick] if you want to simplify this when you're using symbols, you can write `tta_reports: @tta_reports` rather than use a rocket.  We tried to avoid symbols + rockets at my old work but I mean, they work fine so I guess do whatever you want with this suggestion :)